addons/project_todo/static/tests/tours/project_task_activities_split.js

```diff
@@ -0,0 +1,42 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add('project_task_activities_split', {
+    test: true,
+    url: '/web',
+    steps: () => [
+        {
+            content: 'Open Activity Systray',
+            trigger: '.o-mail-ActivityMenu-counter',
+        }, {
+            content: 'Open Task Activities',
+            trigger: '.o-mail-ActivityGroup:contains("Task")',
+        }, {
+            content: 'Task "New Task!" is listed in the activity view',
+            trigger: '.o_activity_record .d-block:contains("New Task!")',
+        }, {
+            content: 'Task "New Sub-Task!" is listed in the activity view',
+            trigger: '.o_activity_record .d-block:contains("New Sub-Task!")',
+            run: () => {
+                if ($(".o_activity_record .d-block:contains(New To-do!)").length !== 0) {
```

Comment:
We should avoid using JQuery. 🙂 
```suggestion
                if (document.querySelectorAll(".o_activity_record .d-block:contains(New To-do!)").length !== 0) {
```